spec/ci/reporter/cucumber_spec.rb

```diff
@@ -121,23 +122,53 @@ def new_instance
           expect(testcases.first).to eql test_case
         end
 
-        it "alters the name of a test case that is pending to include '(PENDING)'" do
+        it "skips and alterss the name of a test case that is pending to include '(PENDING)'" do
```

Comment:
good catchsss. was a typo, don't be ssscared. tsss

![](http://www.esourceengine.com/downloads/minecraft/creeper-tdesign.jpg)